src/Auth/DefaultPasswordHasher.php

```diff
@@ -47,6 +47,7 @@ class DefaultPasswordHasher extends AbstractPasswordHasher
      */
     public function hash(string $password)
```

Comment:
Can we `: string` typehint here? Or at least string|null then?

Reply:
AuthComponents and it's related classes are effectively deprecated so I prefer doing the bare min. to keeps the tools happy.

Reply:
No more hasher? OK, didnt know that.

Reply:
The hashers are included in the new authentication plugin.